### ISSUE TITLE
If https option exists, uses https.

### ISSUE THREAD

**Issue by englercj** [AveVlad/gulp-connect#115]
I'm trying to do this:

```js
    return connect.server({
        root: config.paths.out,
        port: process.env.PORT || 9000,
        https: process.env.HTTPS || false,
        livereload: true,
        middleware: function () {
            return [
                require('connect-gzip').gzip()
            ];
        }
    });
```

But if the `https` option exists it enables https, even if the value is `false` or `undefined`.

**Comment by Shutnik** [comment]
+1

**Comment by Pyo25** [comment]
The problem comes from this line: https://github.com/AveVlad/gulp-connect/blob/master/src/index.coffee#L31
which is translated into plain JS as: `if (opt.https != null) {`

For now you can simply add the property only if the env variable is truthy.
```
var options = {
  root: config.paths.out,
  port: process.env.PORT || 9000,
  livereload: true,
  middleware: function () {
    return [
      require('connect-gzip').gzip()
    ];
  }
}
if (process.env.HTTPS) { options.https = true; }
return connect.server(options);
```

**Comment by justinmchase** [comment]
Fixed in [v3.2.2](https://github.com/AveVlad/gulp-connect/releases/tag/v3.2.2), 404618e1addf819178e4f30a9122c6a3fa8a260a
